builtin/plugin/v5/backend.go

```diff
@@ -72,6 +73,12 @@ func (b *backend) reloadBackend(ctx context.Context) error {
 	}
 	b.Backend = nb
 
+	// Re-initialize the backend in case plugin was reloaded
+	// after it crashed
+	b.Backend.Initialize(ctx, &logical.InitializationRequest{
+		Storage: req.Storage,
+	})
```

Comment:
I want to make sure we're doing the right thing here, but in any case, we should handle the error that could be returned from `b.Backend.Initialize()`.

Reply:
Good call! Error is captured and handled now